README.md

```diff
@@ -695,15 +695,13 @@ This is a list of `THE BEST FOSS apps` according to [us](https://github.com/Psyh
 
 ## Discord
 
-### Aliucord
+### Revenge
```

Comment:
I woudln't replace Aliucord yet, the development has slowed down but not yet to discard this app.